src/style.css

```
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+:root {
+  --main-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  line-height: normal;
+  color: var(--main-color);
+}
+
+.wrap_form {
+  height: 70px;
+}
+
+.wrap_form--small {
+  height: 42px;
+}
```

Is it necessary here?

Check @polosanya comment 

```suggestion
.wrap_form {
  height: 70px;
}

.wrap_form--small {
  height: 42px;
}
```